app/helpers/puppet_related_helper.rb

```diff
@@ -0,0 +1,114 @@
+module PuppetRelatedHelper
+  extend AccessibleResourceHelper
+
+  # initialize puppet related pagelets
```

Comment:
Shouldn't pagelet initialization be done in the initializers? Wouldn't this get executed every time the helper is loaded?